src/apps/dashboard-projects/ProjectsHome.tsx

```diff
@@ -55,18 +55,31 @@ export const ProjectsHome = (props: ProjectsHomeProps) => {
 
   const [sort, setSort] = useState<SortFunction | undefined>();
 
-  const [showProfileNag, setShowProfileNag] = useState(
-    !me.first_name ||
-    !me.first_name.length ||
-    !me.last_name ||
-    !me.last_name.length);
+  const [showProfileNag, setShowProfileNag] = useState(false);
 
   useEffect(() => {
     getMyProfile(supabase).then(({ error }) => {
       if (error) window.location.href = `/${props.i18n.lang}/sign-in`;
     });
   }, []);
 
+  useEffect(() => {
```

Comment:
`me` is coming in as a prop on mount, and wouldn't change I believe. Is there a need for the extra effect hook?

Reply:
Probably my over zealous/SPA bent where I never trust a prop on mount.